### PR TITLE
Invoices

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.2.0
```
Comment (MonsieurSlim) [inline .ruby-version:1]: Are we switching to a higher version of Ruby?

Comment (ghost) [inline .ruby-version:1]: the more current the version, the better. I suppose

File: app/views/layouts/_header.html.haml [hunk 2]
```diff
@@ -10,4 +10,4 @@
       .navbar-text.pull-right
         = succeed "." do
           %strong= current_user.name
-        = link_to "Logout", destroy_user_session_path, method: :delete, class: 'navbar-link'
\ No newline at end of file
+        = link_to(raw('<span class="glyphicon glyphicon-off"></span> Logout'), destroy_user_session_path, method: :delete, class: 'navbar-link')
```
Comment (MonsieurSlim) [inline app/views/layouts/_header.html.haml]: I thought we were using Font awesome, not glyphcons?

File: .ruby-version (hunk 1 above)
Comment (MonsieurSlim) [inline .ruby-version:1]: Cool then.

File: app/views/layouts/_header.html.haml (hunk 2 above)
Comment (ghost) [inline app/views/layouts/_header.html.haml]: Hmm... Will review that then.

Comment (MonsieurSlim) [inline app/views/layouts/_header.html.haml]: That would be `<i class='fa fa-power-off'></i> Log Out` or something similar....

Comment (MonsieurSlim) [inline app/views/layouts/_header.html.haml]: And adding .html_safe should work. I rarely use raw().

Comment (ghost) [inline app/views/layouts/_header.html.haml]: :+1:  cool

File: .ruby-version (hunk 1 above)
Comment (ghost) [inline .ruby-version:1]: I just refactored the project to use both font-awesome and html_safe. please check the latest commit.

**Comment by MonsieurSlim** [comment]
Cool changes. However font awesome uses `<i></1>`tags, not `<span></span>` 
example `<i class="fa fa-bed"></i>`. Here's a list http://fortawesome.github.io/Font-Awesome/icons/

Merge once you make these changes.

**Comment by ghost** [comment]
Done

**Pull request merged**